### ISSUE TITLE
what is your loss function

### ISSUE THREAD

**Issue by connorchen** [abelguima/ryerson-capstone-CKME136#1]
hi @abelguima 
i cant find the loss function you use in your paper, could you share it, thanks.

**Comment by abelguima** [comment]
Hi I use Alexnet with tensorflow in the platform digits.

layer {
  name: "loss"
  type: "SoftmaxWithLoss"
  bottom: "fc8"
  bottom: "label"
  top: "loss"
  exclude { stage: "deploy" }
}

**Comment by abelguima** [comment]
Install nvidia Digits in your computer.

**Comment by connorchen** [comment]
thanks for your reply@abelguima
The probability of each position does not affect each other, maybe Logistic loss will be better, and thank you for letting me know Digits , which is great.
by the way the accuracy in your paper is train accuracy or validation accuracy.
